float64[] center_position
float64 orientation
float64[] axes_length
bool is_boundary